packages/@uppy/core/src/Uppy.ts

```diff
@@ -1542,10 +1536,6 @@ export class Uppy<M extends Meta, B extends Body> {
     }
 
     this.on('upload-start', (files) => {
```

Comment:
Do we need the arrow function?
```suggestion
    this.on('upload-start', onUploadStarted);
```